Dockerfile.fpm

```diff
@@ -1,20 +1,21 @@
 FROM elifesciences/php_fpm:20180129144019
 USER root
 RUN docker-php-ext-install pcntl
+# TODO: move in base image
+RUN apt-get update && apt-get install libfcgi0ldbl
+COPY curl_fpm /srv/bin/
 
+ENV PROJECT_FOLDER=/srv/annotations
 WORKDIR /srv/annotations
 COPY composer.json composer.lock /srv/annotations/
-# could probably use --no-dev and other optimizations?
-RUN composer install
+RUN composer --no-interaction install --no-suggest --no-dev
```

Comment:
`--classmap-authoritative` as well?

Reply:
It may not break `dev`, because https://github.com/elifesciences/annotations/blob/develop/docker-compose.yml#L20 will mount the whole folder (including `vendor/`) inside the container.